app/graph/types/query_type.rb

```diff
@@ -140,9 +125,8 @@
     end
   end
 
-  field :dynamic_annotation_field do
-    type DynamicAnnotationFieldType
-
+  # TODO Remove this
+  field :dynamic_annotation_field, DynamicAnnotationFieldType, 'Query an annotation field' do
```

Comment:
Just to note - when we remove this, we need to move the `only_cache` and `query` arguments which are pretty important - this is a way to get an annotation field by searching inside the JSON column